### PR TITLE
last/welcome.php is fixed. Running last link

### PR DESCRIPTION
**Pull request by josephwithay**
Signed-off-by:josephwithay yoseph_jo@hotmail.com

### CODE REVIEW

**Comment by kennethtran93** [comment]
Sorry, but please pull latest team repo and repush.

**Comment by josephwithay** [comment]
Why are there conflicts??

File: application/controllers/last/Welcome.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Our homepage. Show a table of all the author pictures. Clicking on one should show their quote.
+ * Our quotes model has been autoloaded, because we use it everywhere.
+ * 
+ * controllers/Welcome.php
```
Comment (kennethtran93) [inline application/controllers/last/Welcome.php]: it's controllers/last/Welcome.php

File: application/controllers/last/Welcome.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		parent::__construct();
+	}
+
+	//-------------------------------------------------------------
+	//  The normal pages
+	//-------------------------------------------------------------
+
+	function index()
+	{
+		$this->data['pagebody'] = 'homepage';	// this is the view we want shown
```
Comment (kennethtran93) [inline application/controllers/last/Welcome.php]: this is not right...should be justone

**Comment by kennethtran93** [comment]
Where are the changes for last/Welcome.php?

**Comment by kennethtran93** [comment]
From new branch: Good
Signed: Good enough...
Formatting: Good
Changelog: Good
Does it work: Yes.

PR Approved

**Pull request merged**